cms_backend/settings.py

```diff
@@ -32,13 +32,12 @@
 
 INSTALLED_APPS = [
     'django.contrib.admin',
-    'users.apps.UsersConfig'
```

Comment:
Dont remove `users.apps.UsersConfig` as of now.

Reply:
but, the command to run django runsrever failed

see the attached SS  below:
![image](https://user-images.githubusercontent.com/60546840/104282536-11fa2100-5464-11eb-9f31-dad7dd29deb8.png)

Reply:
The problem is you haven't registered the `users` app under `INSTALLED_APPS` in `settings.py`. You'll have to register each app you create inside `INSTALLED_APPS`. Add `users.apps.UsersConfig` inside installed apps.

Reply:
ohk got it